src/configuration.ts

```diff
@@ -335,6 +335,7 @@ const baseDefaults: IBaseConfiguration = {
   skipFiles: [],
   smartStep: true,
   sourceMaps: true,
+  // keep in sync with sourceMapPathOverrides in package.json
```

Comment:
@roblourens do you have thoughts on perhaps generating the `contributes` section of the package.json? We're passing it through a Gulp build anyway, injection would be feasible. We have a bunch of common options duplicated and defaults we need to make sure to keep in sync, automating it would make it cleaner (and also let us create it in a type-safe way, which would be neat)

Reply:
I'm not really a fan. Once it's set up, the contributions almost never change, so it seems like we should prefer simplicity.